File: Bimestre 1/Java/Estructuras Condicionales/estructuras_condicionales/src/estructuras_condicionales/Conversion_Graddos.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package estructuras_condicionales;
import java.util.Scanner;
/**
 *
 * @author diegojp
 */
public class Conversion_Graddos {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        //Declaracion de variables
        Scanner leer = new Scanner(System.in);
        double gC, gF,gK;
        gC = 0; gF = 0; gK = 0;
        
        //entrda de datos
        System.out.println("Estructuras logicas condicionales simples");
        System.out.println("Programa para convertir en grados \n");
        
        System.out.println("Ingrese los grados centigrados: ");
        gC = leer.nextDouble();
        
        if (gC > 0){
            gF =(9/5)*(gC + 32);
            gK = gC + 273.15;
        System.out.println("La equivalencia en grados Farenheid es "+gF);
        System.out.println("La equivalencia en grados Kelvin es "+gK);
        }
        
        
    }
    
}

File: Bimestre 1/Python/estructuras_condicionales/Conversion_Graddos.py
gC = 0; gF = 0; gK = 0;

#entrda de datos
print("Estructuras logicas condicionales simples")
print("Programa para convertir en grados \n")

print("Ingrese los grados centigrados: ")
gC = float(input())

if (gC > 0):
    gF =(9/5)*(gC + 32)
    gK = gC + 273.15
print("La equivalencia en grados Farenheid es ",gF)
print("La equivalencia en grados Kelvin es ",gK)
